sdk/python/kfp/_client.py

```diff
@@ -123,10 +125,25 @@ def _load_config(self, host, client_id, namespace, other_client_id, other_client
 
     token = None
 
-    # Obtain the tokens if it is IAP or inverse proxy.
-    # client_id is only used for IAP, so when the value is provided, we assume it's IAP.
-    if client_id:
-      token = get_auth_token(client_id, other_client_id, other_client_secret)
```

Comment:
Why is this branch removed?

Reply:
my mistake when merging codes, fixed. thanks